packages/contracts-bedrock/src/L1/LeaderElectionBatchInbox.sol

```diff
@@ -5,6 +5,40 @@ pragma solidity ^0.8.0;
 /// @notice Interface for implementing a leader election scheme
 
 abstract contract LeaderElectionBatchInbox {
+    enum LeaderStatusFlags {
+        Scheduled,
+        Unscheduled,
+        Invalid
+    }
+
+    struct MetaData {
```

Comment:
No need to do it for now, by we may add a comment about the need for some commitment/hash to the actual data stored somewhere else, so that the derivation pipeline can check the consistency between metadata and data.

Reply:
I added some comments to the submit function. Does this cover what you had in mind?

https://github.com/EspressoSystems/op-leader-election/blob/61df6481a82cd324aac9bfcdb2fb007c475418cf/packages/contracts-bedrock/src/L1/LeaderElectionBatchInbox.sol#L26-L27